src/tito/release/distgit.py

```diff
@@ -387,6 +395,98 @@ class DistGitReleaser(FedoraGitReleaser):
     cli_tool = "rhpkg"
 
 
+class DistGitMeadReleaser(DistGitReleaser):
+    REQUIRED_CONFIG = ['mead_scm', 'branches']
+
+    def __init__(self, name=None, tag=None, build_dir=None,
+        config=None, user_config=None,
+        target=None, releaser_config=None, no_cleanup=False,
+        test=False, auto_accept=False,
+        prefix="temp_dir=", **kwargs):
+
+        if 'builder_args' in kwargs:
+            kwargs['builder_args']['local'] = False
+
+        DistGitReleaser.__init__(self, name, tag, build_dir, config,
+                user_config, target, releaser_config, no_cleanup, test,
+                auto_accept, **kwargs)
+
+        self.mead_scm = self.releaser_config.get(self.target, "mead_scm")
+
+        self.mead_url = "%s#%s" % (
+            self.mead_scm,
+            self.builder.build_tag)
+        self.push_url = self.releaser_config.get(self.target, "mead_push_url")
+
+    def _sync_mead_scm(self):
+        with chdir(self.git_root):
+            print("Syncing local repo with %s" % self.push_url)
+            try:
+                run_command("git push %s %s" % (self.push_url, self.builder.build_tag))
+            except RunCommandException as e:
+                if "rejected" in e.output:
+                    if self._ask_yes_no("The remote rejected a push.  Force push? [y/n] ", False):
+                        run_command("git push --force %s %s" % (self.mead_scm, self.builder.build_tag))
```

Comment:
Hmm what's the differentiation between mead_scm and mead_push_url, and why do we switch from one to the other here when the user specified a force push?

Should mead_push_url be marked as a required param as well?

We should get these and the whole builder documented in doc/builders.mkd.